src/ncp/spinel.h

```diff
@@ -1728,6 +1729,37 @@ typedef enum {
      */
     SPINEL_PROP_TIME_SYNC_XTAL_THRESHOLD = SPINEL_PROP_OPENTHREAD__BEGIN + 9,
 
+    /// Child Supervision Interval
+    /** Format: `S` - Read-Write
+     *  Units: Seconds
+     *
+     * Required capability: `SPINEL_CAP_CHILD_SUPERVISION`
+     *
+     * The child supervision interval (in seconds). Zero indicates that child supervision is disabled.
+     *
+     * When enabled, Child supervision feature ensures that at least one message is sent to every sleepy child within
```

Comment:
This is only applicable for FTDs, right?

Reply:
Yes. Let me change this properties' scope to be only under FTD build.